eppo_client/config.py

```diff
@@ -1,5 +1,5 @@
 from eppo_client.assignment_logger import AssignmentLogger
-from eppo_client.base_model import SdkBaseModel
+from eppo_client.models import SdkBaseModel
```

Comment:
I don't love this – it works because `models` itself imports `SdkBaseModel`, but it feels very brittle (removing or changing that import in models would break things unexpectedly). Why not import directly from `base_model` here?

Reply:
Good catch, this was unintentional